client/src/store/reducers/ui/__tests__/typography-test.js

```diff
@@ -81,3 +100,69 @@ describe('store/reducers/ui/typography/decrementFontSize', () => {
     expect(state).to.deep.equal(initialState);
   });
 });
+
+describe('store/reducers/ui/typography/incrementMargins', () => {
+  it('Should increment the value by 1', () => {
+    const initialState = {
+      margins:{
+        current: 1,
+        max: 2
+      }
+    };
+
+    const action = { type: 'INCREMENT_MARGINS' };
+    const state = typographyReducer(initialState, action);
+    expect(state).to.deep.equal({
+      margins:{
+        current: 2,
+        max: 2
+      }
+    });
+  });
+
+  it('Should return the same value if it is at maximum', () => {
+    const initialState = {
+      margins:{
+        current: 2,
+        max: 2
+      }
+    };
+
+    const action = { type: 'INCREMENT_MARGINS' };
+    const state = typographyReducer(initialState, action);
+    expect(state).to.deep.equal(initialState);
+  });
+});
+
+describe('store/reducers/ui/typography/decrementFontSize', () => {
+  it('Should decrement the value by 1', () => {
+    const initialState = {
+      margins:{
+        current: 1,
+        min: 0
+      }
+    };
+
+    const action = { type: 'DECREMENT_MARGINS' };
+    const state = typographyReducer(initialState, action);
```

Comment:
'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).